freqtrade/exchange/common.py

```diff
@@ -102,7 +102,7 @@ def calculate_backoff(retrycount, max_retries):
 def retrier_async(f):
     async def wrapper(*args, **kwargs):
         count = kwargs.pop('count', API_RETRY_COUNT)
-        kucoin = args[0].name == "Kucoin"  # Check if the exchange is KuCoin.
+        kucoin = args[0].name.lower() == "kucoin"  # Check if the exchange is KuCoin.
```

Comment:
It's interesting that this was not noticed before ... as this effectively disables this fix completely.

considering the frequency this will be called, i think we should prefer to use the proper casing in the string instead.
That'll avoid converting a static (!) string to lowercase whenever we fetch ohlcv data.